sentry-ruby/spec/sentry/event_spec.rb

```diff
@@ -107,6 +107,30 @@
           cookies: {}
         )
 
+        expect(event.to_hash[:tags][:request_id]).to eq("abcd-1234-abcd-1234")
+        expect(event.to_hash[:user][:ip_address]).to eq("2.2.2.2")
+      end
+    end
+
+    context "with config.trusted_proxies = [\"2.2.2.2\"]" do
```

Comment:
this block should be placed inside the `"with config.send_default_pii = true"` context.  and then you won't need to set `Sentry.configuration.send_default_pii = true` again in the before block.

Reply:
I _think_ I understood. Let me know if I was mistaken 👍